concent_api/common/singleton.py

```diff
@@ -1,15 +1,50 @@
+from typing import Callable
 from django.conf        import settings
-from golem_sci.factory  import new_sci_rpc
+from ethereum.transactions import Transaction
+from golem_sci.client import Client
+from golem_sci.contracts.provider import ContractDataProvider
+from golem_sci.implementation import SCIImplementation
+from golem_sci.transactionsstorage import TransactionsStorage
+
+from web3 import HTTPProvider
+from web3 import Web3
+from web3.middleware import geth_poa_middleware
+
+from core.payments.storage import DatabaseTransactionsStorage
+
+
+#  Temporary SCI factory for Concent needs.
+#  Gets storage as a parameter
+def concent_sci(
```

Comment:
Might be a good idea to say in a comment where the original code came from.

Reply:
Done